test/lib/MangroveTest.sol

```diff
@@ -156,7 +156,8 @@ contract MangroveTest is Test2, HasMgvEvents {
     quote: TokenOptions({name: "Quote Token", symbol: "$(B)", decimals: 18}),
     defaultFee: 0,
     gasprice: 40,
-    gasbase: 50_000,
+    //FIXME: measure gasbase and consider using different measurement method (see GasBaseTest.t.sol)
+    gasbase: 150_000,
```

Comment:
use different measurement method compared to what?

Reply:
We currently use transfers inside ActivateSemibook - IMHO this is wrong, we should use the estimate given by GasBaseTest.
(Even gasBaseTest is not the worst case, that would be with a failing posthook, but since failing posthooks does not happen every single time it may be good).

Reply:
Ok. Did you run the GasBaseTest and set the parameter based on that?

Reply:
Yes, but unoptimized

Reply:
Ok, makes sense.

So the FIXME is for having this discussion, right? Or is there more to it?

Maybe it could be changed into a comment describing how to check/update `gasbase` for `MangroveTest`?

Reply:
I have updated it with a newly measured value. Note that this is for an entire market order of 1 offer to an empty book. Previously we used only transfers as baseline which would give a value of 128176.

Reply:
Added FIXME in ActivateSemibook